src/encoded/schemas/sc_atac_alignment_quality_metric.json

```diff
@@ -180,7 +180,8 @@
                     "title": "MIME type",
                     "type": "string",
                     "enum": [
-                        "text/plain"
+                        "text/plain",
```

Comment:
From the ticket description, should text/plain be removed completely? Do we want to allow both?

Reply:
Decided to allow both, approved.